docs/Rollup_basics.md

```diff
@@ -13,42 +13,45 @@ and lower cost than on-chain Ethereum contracts.
+> Note: Earlier versions of Arbitrum did not fully support dynamic contract launching and required code for all contracts to be provided when initially launching the chain. We have now removed this limitation.
+
+Applications that reside on the same ArbChain can make synchronous calls to one another exactly as they would on Ethereum.
+Typically, you'll want to group dApps together into a single ArbChain if they need to interact and compose with one another. Applications that do not require interaction with others can be deployed on their own ArbChain.
 
 Every ArbChain has a set of Validators, who monitor the chain to ensure that it executes correctly.
-Validators will deposit currency stakes, which they will lose if they behave dishonestly.
+Validators will deposit currency stakes, which they will lose if they behave dishonestly. Validating an Arbitrum Rollup chain is permissioneless; anyone who puts down a stake can serve as a validator.
 
 An ArbChain contains a set of contracts.
-Over time, you can start new contracts in an ArbChain, and contracts can self-destruct.
-There is one limitation, though: you have to provide the code for a contract when you initially build the chain.
-You can start as many instances of those contracts as you want,
-but you can't introduce new contract code into a chain after the chain has started.
-(This limitation will probably be lifted in future versions.)
+Over time, you can launch new contracts in an ArbChain, and contracts can self-destruct, exactly as they do on the Etherum blockchain.
+ArbChains accept EVM transactions that are byte-for-byte identical to Ethereum contracts.
+
 
 ## How to make an ArbChain
 
-There are two steps required to start an ArbChain.
 
-First, you compile your Solidity contracts, using the open-source Arbitrum compiler.
-This will bundle your contracts together, along with some runtime and "glue" code,
-into an Arbitrum executable file (or ".ao file").
-The .ao file is a single program, to be run on the Arbitrum Virtual Machine (AVM) architecture,
-which will manage and run the contracts and wallets needed for your chain.
-Think of it as an ArbChain in a box.
+Suppose you’re a developer who has written a dapp for the Ethereum platform. Arbitrum interoperates with Ethereum, so you can launch your dapp on an Arbitrum Rollup chain and get better speed and scalability. 
+
+You’re starting with a dapp—or you’re planning to develop one—that’s made up of some contracts written in Solidity, along with a browser-based front end.  Here’s how to use Arbitrum with your dapp. Arbitrum supports dynamic launching of contracts on deployed chains using Ethereum's CREATE and CREATE2 opcodes, so you can either launch a new Rollup chain for your dapp or deploy it to an existing one. Here, we describe the process for launching a new chain.
```

Comment:
As above, can deploy contracts using CREATE and CREATE2, but also using transactions to address zero, just like Ethereum.  Standard build tools use the address zero tx method, so we should emphasize that more here.  Should also say that you can use standard build tools like Truffle (and Buidler?)